js/views/conversation_view.js

```diff
@@ -15,6 +15,11 @@
             return { toastMessage: i18n('unblockToSend') };
         }
     });
+    Whisper.LeftToast = Whisper.ToastView.extend({
```

Comment:
`LeftGroupToast` might be a little less ambiguous - when I first read this, I thought 'left side of the screen'